AUTHORING_GUIDE.md

```diff
@@ -704,12 +704,12 @@ commonly for APIs that read files from GCS. To run the tests for these
 samples, configure your GCS bucket name via the `CLOUD_STORAGE_BUCKET`
 environment variable.
 
-The resources required by tests can usually be found in the `./resources`
-folder inside the sample directory, as in [this
-example](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/automl/cloud-client/resources).
+The resources required by tests can usually be found in the `./samples`
+folder inside the respective products, as in [this
+example](https://github.com/googleapis/python-pubsub/tree/master/samples).
 You can upload those resources to your own GCS bucket to run the tests with
 [gsutil](https://cloud.google.com/storage/docs/gsutil). For example:
 
 ```console
-gsutil cp ./resources/* gs://$CLOUD_STORAGE_BUCKET/
+gsutil cp ./samples/* gs://{$CLOUD_STORAGE_BUCKET}/
```

Comment:
I see couple of things.

I took a quick look at python-pubsub repo. It has `samples/snippets/resources` directory.
I think we should only upload the `resources` subdirectory (no need to upload the sample code etc).

Maybe 
```
gsutil cp ./samples/snippets/resources gs://{$CLOUD_STORAGE_BUCKET}/
```

Also, tests in python-pubsub repo are only used locally(I may be wrong), so I'd stick with automl sample instead of python-pubsub.

Reply:
Done! I didn't notice the resources in the samples directories, thanks for catching that.

Updated the rest accordingly.

Reply:
Or maybe:

```
gsutil cp ./samples/snippets/resources/* gs://{$CLOUD_STORAGE_BUCKET}/
```

It depends on what path that the tests are expecting.